Line/README.md

```diff
@@ -0,0 +1,68 @@
+# Use LINE Login with Firebase
+
+This sample shows how to Sign in Firebase using [LINE Login](https://developers.line.me/line-login/overview).
+
+## Setup the sample
+
+### LINE business account's creation and setup
+
+ 1. Follow the instruction in [LINE Developers document](https://developers.line.me/line-login/overview) to setup a LINE business account.
+ 1. After finished setting up your channel, follow the document to integrate LINE SDK in your [iOS](https://developers.line.me/ios/overview) and [Android](https://developers.line.me/android/overview) app. It should include:
+  * Open **Technical Configuration** page of your LINE channel, and update the following values:
+    * iOS Bundle ID: `com.google.firebase.LINELoginDemo`
+    * Android Package Name: `com.google.firebase.linelogindemo`
+    * Android Package Signature: The SHA1 value of your keystore, without semicolons
+  * Download the LINE SDK for iOS and Android from LINE developers site and add them to your apps. The latest version of LINE SDK may have different API with that used to create the sample app, so you may need to make some changes to the sample apps. LINE SDK versions being used in building the sample apps are:
+    *  iOS: **3.2.1**
+    *  Android: **3.1.20** 
+  * Update your Channel ID to your apps
+    * iOS: Update `<your_channel_id>` in `LineAdapter.plist`
+    * Android: Update `<your_channel_id>` in `app/build.gradle` 
```

Comment:
I have to put it in gradle file because I need it to concatenate a config value for LINE SDK in Android.manifest. 

Please let me know if there's a simple way to set gradle variable using resource config. I'm happy to put config value in configs.xml :)